mex/editor/rules/main.py

```diff
@@ -234,17 +234,34 @@ def additive_rule_input(
         ),
         rx.cond(
             input_config.editable_text,
-            rx.input(
-                placeholder="Text",
-                value=value.text,
-                on_change=RuleState.set_text_value(field_name, index),
-                style={
-                    "margin": "calc(-1 * var(--space-1))",
-                    "width": "100%",
-                },
-                custom_attrs={
-                    "data-testid": f"additive-rule-{field_name}-{index}-text"
-                },
+            rx.cond(
+                input_config.render_textarea,
+                rx.text_area(
+                    placeholder="Text",
+                    value=value.text,
+                    on_change=RuleState.set_text_value(field_name, index),
+                    style={
+                        "margin": "calc(-1 * var(--space-1))",
+                        "width": "100%",
+                    },
+                    custom_attrs={
+                        "data-testid": f"additive-rule-{field_name}-{index}-text"
```

Comment:
```suggestion
                        "data-testid": f"additive-rule-{field_name}-{index}-textarea"
```

would make it easier for the tests to know what is rendered

--- 

could you also add a test, analogous to `test_edit_page_renders_text_input`, for the textarea please?

Reply:
I dont think this is a good idea. I understood it that way, that tests wanna find the correct control for clicking, filling, etc and the "_text" suffix indicates that its an text value (like link or identifier). The other main downfall is, that the tests then have to know what kind of field is rendered when they wanna fill in text => leads to alot of changes in existing tests and also every test needs to read the model.yaml to generate the _text or _textarea suffix (or u read it and write it static). Therefore i disagree with the suggested change.

Reply:
alright, fair points